qir/qsharp/HiddenShift/Makefile

```diff
@@ -1,7 +1,7 @@
 all: qat-qsharp
 
 qat-qsharp: QSharpVersion/qir/Example.ll
-	${QAT_BINARY} --apply -S -O3 --verify-module --validate --always-inline --unroll-loops --profile default QSharpVersion/qir/Example.ll  || echo "failed!"
+	${QAT_BINARY} --apply -S --verify-module --validate --always-inline --unroll-loops --adaptor default QSharpVersion/qir/Example.ll  || echo "failed!"
```

Comment:
Repeating fragment `${QAT_BINARY} --apply -S --verify-module --validate --always-inline --unroll-loops` in a number of files. Consider de-duplicating.

Reply:
I think this would be part of a larger refactor: We should rearrange the examples and move to a more unified build system. I've created an issue for this item https://github.com/qir-alliance/qat/issues/140 .